tests/react/async.test.tsx

```diff
@@ -958,15 +966,18 @@ it('async atom double chain without setTimeout (#751)', async () => {
     </StrictMode>
   )
 
+  await findByText('loading')
   await findByText('async: init')
 
-  fireEvent.click(getByText('button'))
+  await userEvent.click(getByText('button'))
   await findByText('loading')
   resolve()
   await findByText('async: ready')
 })
 
 it('async atom double chain with setTimeout', async () => {
+  vi.useFakeTimers({ shouldAdvanceTime: true })
```

Comment:
Using fake timers in this test seems against the purpose of the test.

Reply:
I was hoping to advance the time using fake timers 😅 but you're right, it's more of a hack. I'll see if I could remove these hacks altogether 🤔